### PR TITLE
Styleguide Basic Components

### PR DESCRIPTION
**Pull request by XeniaRud**

### CODE REVIEW

File: src/frontend/src/components/CheckBox/CheckBox.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+				tabIndex={0}
+				data-label={label}
+				//checked={this.state.checked}
+			/>
+			<p className="label" data-label={label}>
+				Label Name
+			</p>
+		</div>
+	);
+};
```
Comment (duci9y) [inline src/frontend/src/components/CheckBox/CheckBox.tsx]: Did you run Prettier on this file?

File: src/frontend/src/components/Link/Link.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+	return (
+		<div>
+			<nav>
+				<a href="https://google.com">GOOGLE LINK</a>
+			</nav>
+		</div>
+	);
+};
+
+export default Link;
```
Comment (duci9y) [inline src/frontend/src/components/Link/Link.tsx]: This file needs Prettier!

File: src/frontend/src/components/TextField/TextField.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+// class TextField extends React.Component {
+
+// function TextField() {
+// 	return (
+// 		<View>
+// 			<Image alt="alt text goes here" src="exampleImage.png" />
+// 		</View>
+// 	);
+// }
+
```
Comment (duci9y) [inline src/frontend/src/components/TextField/TextField.tsx]: You should get rid of these comments if you don't want them.
```suggestion

```

File: src/frontend/src/components/CheckBox/CheckBox.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      <input
+        className={
+          checked ? 'Checkbox-container checked' : 'Checkbox-container'
+        }
+        onClick={handleClick}
+        role="button"
+        type="checkbox"
+        tabIndex={0}
+        data-label={label}
+        //checked={this.state.checked}
```
Comment (nivi-nath) [inline src/frontend/src/components/CheckBox/CheckBox.tsx]: remove commented out code before merge

**Pull request merged**